app/util/api/jira_clients.py

```diff
@@ -195,3 +195,8 @@ def get_locale(self):
         api_url = f'{self.host}/rest/api/2/myself'
         user_properties = self.get(api_url, "Could not retrieve user")
         return user_properties.json()['locale']
+
+    def get_applications_properties(self):
+        api_url = f'{self.host}/rest/api/2/application-properties'
+        user_properties = self.get(api_url, "Could not retrieve user")
```

Comment:
`Could not retrieve applications properties`

Reply:
Thx! Three times I've made changes and the last one forgot